data-serving/data-service/schemas/cases.schema.json

```diff
@@ -2,12 +2,6 @@
   "$jsonSchema": {
```

Comment:
Do we have any notion of "this is the stuff we're currently importing"? Is it basically look at what's in vs. out of importedCases or the python file?

Reply:
right now it's not documented anywhere -- I'll do my best to document this in the next PR.

right now I'm including everything that's nonempty in importedCases, even if we converted it. may change this -- not sure yet